src/components/Reports/DesignationAccountsReport/SetActiveDesignationAccount.graphql

```diff
@@ -0,0 +1,13 @@
+mutation SetActiveDesignationAccount(
+  $input: SetActiveDesignationAccountInput!
+) {
+  setActiveDesignationAccount(input: $input) {
+    active
+    balanceUpdatedAt
+    convertedBalance
+    currency
+    designationNumber
+    id
+    name
```

Comment:
Does this need all these fields? I assume `active` is the only thing that would change. And we probably need `id` to make the cache update work.

Reply:
Basically I used all fields while trying to get the `DesignationAccount` type so the cache can be automatically updated.

Reply:
If the response type is not `DesignationAccount`, the cache didn't update.

Reply:
Right. I was thinking the server absolutely needs to provide all these fields and use the `DesignationAccount` type. But I'm not sure the client actually needs to request all those fields, besides `id` and `active`. But if it's needed, it's not hurting anything 🤷‍♂️